_posts/0000-01-02-sapna-jadhav.md

```diff
@@ -0,0 +1,6 @@
+--
+layout: slide
+title: "Welcome to our second slide!"
+---
```

Comment:
Welcome to your repository for your GitHub Learning Lab course. This repository will be used during the different activities that I will be guiding you through. See a word you don't understand? We've included an emoji 📖 next to some key terms. Click on it to see its definition.